aws-android-sdk-mobile-client/src/androidTest/java/com/amazonaws/mobile/client/AWSMobileClientTestBase.java

```diff
@@ -49,4 +50,9 @@ public static String getValidJWT(long expiryInSecs){
         String validAccessToken = accessToken_p1_Base64+"."+accessToken_p2_Base64+"."+accessToken_p3_Base64;
         return validAccessToken;
     }
+    @Test
+    public void empty()
```

Comment:
Document that this is need for CircleCI to run the test suite.

Reply:
done